### ISSUE TITLE
Not working for 2018.3.3 :(

### ISSUE THREAD

**Issue by marcusorjames** [gugglegum/phpstorm-reset-trial#2]
Error: Directory ~/.PhpStorm2018.3/config is not looks like valid PhpStorm config directory

**Comment by gugglegum** [comment]
@marcusorjames Could you please provide more information? Which operating system you use? Did you set different config directory? Is this directory exist? If yes, what it contains?

**Comment by marcusorjames** [comment]
Ubuntu 18.04

**tldr: the PropertiesComponent xml node is no longer in options.xml it's in other.xml I will pull request**

Ok I've had a deeper look at this, the code checks for a file options.xml. I have tried other ways to get this to reset and it included deleting this file.

I tried to restore it from a previous version so that it exists and the resetter worked. It did so much except it again didn't create options.xml so I had to abort.

The problem I am getting is that now every time I start up PHPStorm it chucks you out after 1 minute rather than the 30 (like it knows I tried to bypass it). When it kicks you out it also deletes the bloody options.xml file... again recreating the file and running the resetter exactly the same thing happens

...
So it turns out the PropertiesComponent has moved to other.xml, so I guess this needs changing. I will see if I can fix it and do a pull request.

**Comment by marcusorjames** [comment]
#3 Here we go :) Just rename options.xml to other.xml and boom 30 days

**Comment by gugglegum** [comment]
@marcusorjames Thank you for you pull request. I'll review it little later. At now I see that this PR breaks backward compatibility with PhpStorm2018.3.2 and lower. Need to make universal solution to detect valid file with options.

**Comment by marcusorjames** [comment]
Yeh it does sorry for being lazy I guess we need to store, which file the
xml node is in in a variable set by detecting where it can be read from.

**Comment by gugglegum** [comment]
@marcusorjames Yes, exactly. Also I want to investigate how PhpStorm 2018.3.3 changes config folder on upgrade. It copies or renames xml file? Is it happens on first run or durring install? I think we need to use other.xml if exists and only if not -- use options.xml. Currently I still didn't upgraded to 2018.3.3, still on 2018.3.2. I'll check how it's happening.
